piet-cairo/src/lib.rs

```diff
@@ -39,6 +44,7 @@ impl<'a> CairoRenderContext<'a> {
         CairoRenderContext {
             ctx,
             text: CairoText::new(),
+            transform_stack: vec![Affine::default()],
```

Comment:
wow now I'm doubting my own implementation. I think this should be an empty vec?? because a call to `restore` without a call to `save` should error.

This was expected below, in `transform`, where we check if there's an item in the stack, as well as in `current_transform` where we do an `unwrap_or_default` 🤔

Does my thinking make sense to you?

Reply:
Indeed, this should definitely be empty. I pushed the change.